build/diva/install.sh

```diff
@@ -0,0 +1,14 @@
+echo "---- INSTALL Leaflet  ----"
+
+divabranch=master
+
+# cleanup if it exists
+rm -rf diva
+
+# get mirador
+git clone git://github.com/DDMAL/diva.js.git || { echo "diva git clone failed" ;exit 1; }
```

Comment:
Forgot to submit this comment, but let's consider changing this to the git command mentioned in one of the other comments. Having a folder with the `.js` extension feels weird so I recommend cloning into `divajs`.